python/tvm/relay/frontend/pytorch.py

```diff
@@ -2714,7 +2714,7 @@ def _run_jit_passes(graph):
     """ The inline pass is necessary to unwrap prim::CallMethod """
     import torch
 
-    if _is_version_greater_than("1.5.0"):
+    if _is_version_greater_than("1.5.1"):
```

Comment:
This fix will not work if the version string is of the form "1.5.1+cpu" or the like. The method _is_version_greater_than will return "1.5.1+cpu" is greater than "1.5.1" since that is what packaging.version.parse returns